frontend/src/components/elements/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import { useTheme } from "emotion-theming"
+import { Theme } from "src/theme"
+import { Spinner as SpinnerProto } from "src/autogen/proto"
+import AlertContainer, { Kind } from "src/components/shared/AlertContainer"
+import StreamlitMarkdown from "src/components/shared/StreamlitMarkdown"
+import {
+  StyledSpinnerContainer,
+  ThemedStyledSpinner,
+} from "./styled-components"
+
+export interface SpinnerProps {
+  width: number
+  element: SpinnerProto
+}
+
+function Spinner({ width, element }: SpinnerProps): ReactElement {
+  const theme: Theme = useTheme()
+
+  return (
+    <div className="stSpinner">
+      <AlertContainer width={width} kind={Kind.INFO}>
```

Comment:
I think we can lose the AlertContainer altogether. I did it on my end and it looks great!

Reply:
Yep, that would be no problem to drop the `AlertContainer` :) Maybe the only thing to consider is that the alert framing is closer to the look & feel of the current spinner, so it could be slightly visually unexpected for existing apps. However, I think that is only a very minor issue.

Reply:
Yeah, I wondered about that too, but decided we need to be able to update how things look incrementally or we'd be stuck with weird UIs for a long time.

Plus every app I tried looked better without it 😉

Reply:
Alright, sounds good :) I will remove the `alert` container.

Reply:
I pushed some commits to remove the `alert` container today. If this state is fine for you, I will merge it in.